steps/test/unit/tReorderMSData.cc

```diff
@@ -1,94 +1,115 @@
 
-#include <fstream>
 #include <algorithm>
 #include <boost/test/unit_test.hpp>
+#include <fstream>
+#include <limits>
+
+#include "dp3/steps/Step.h"
 
 #include "tStepCommon.h"
-#include <dp3/steps/Step.h>
-#include "../../NullStep.h"
+
+#include "../../../common/ParameterSet.h"
+
 #include "../../InputStep.h"
+#include "../../NullStep.h"
 #include "../../ReorderMSData.h"
-#include <dp3/base/DPBuffer.h>
-#include <dp3/base/DPInfo.h>
-#include "../../../common/ParameterSet.h"
-#include "../../../common/StringTools.h"
 
-using dp3::base::DPBuffer;
-using dp3::base::DPInfo;
 using dp3::common::Fields;
 using dp3::common::ParameterSet;
 using dp3::steps::Step;
 
 BOOST_AUTO_TEST_SUITE(reorder)
 
-bool compareBinaries(const std::string& filename1,
-                     const std::string& filename2) {
-  std::ifstream file1(
-      filename1,
-      std::ifstream::ate | std::ifstream::binary);  // open file at the end
-  std::ifstream file2(
-      filename2,
-      std::ifstream::ate | std::ifstream::binary);  // open file at the end
+// Function to compare two binary files. If the files for comparison are Meta
+// file, the flag isMetaFile is set to true.
+void compareBinaryFiles(const std::string& reference_filename,
+                        const std::string& input_filename, bool isMetaFile) {
```

Comment:
`isMetaFile` does not follow variable naming conventions.
Please read over the entire style guideline for variable naming conventions and then check that all variables comply, pay attention also to rules about abbreviations etc.

See yesterdays comment:


For new code (like everything in this file) please adhere to naming conventions from google c++ style guide https://google.github.io/styleguide/cppguide.html#Variable_Names

e.g. input_file not inputFile please pay attention also to the rule of `Minimize the use of abbreviations` while reading over the style guide.

If modifying existing functions that use old conventions then you can in those cases (at your discretion, if it makes sense) stick to the existing style within those functions for consistency instead.

Reply:
Changed the variable name to "is_meta_file". I have changed all the variables names to follow snake-cases.

Reply:
You've missed the rest of the comment about abbreviations though, please read over the entire variable names portion of the link